cypress/PageObjects/AuthenticationPage.js

```diff
@@ -0,0 +1,48 @@
+export class AuthenticationPage {
+
+    //Locators 
+    getURLtext() {
```

Comment:
make naming for getters upper-case

Reply:
Do you mean instead of  camelCase like 'getLoginEmailInput()' use 'GETLOGINEMAILINPUT() ?

Reply:
GET_LOGIN_EMAIL_INPUT_SELECTOR

like this